components/TextFields/src/MDCTextInput.h

```diff
@@ -48,6 +49,21 @@
 @property(nonatomic, nullable, copy) NSAttributedString *attributedText;
 
 /**
+ The path of the area to be highlighted with a border. This could either be with a drawn line or a
+ drawn fill.
+
+ Note: The settable propertys of the UIBezierPath are respected (.lineWidth, etc).
```

Comment:
Checked with Apple docs. They do indeed spell it your way: [properties](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/EncapsulatingData/EncapsulatingData.html).